devlib/native/win_native.cpp

```diff
@@ -304,18 +306,17 @@ namespace winutil {
     }
 
 
-    static auto getDevInfo(const QString containerId) -> DevInfo {
-        DevInfo devInfo;
+    static auto getUsbDevicesByContainerIdsMap(void) -> QMap<QString, DeviceProperties> {
+        auto usbDevicesByContainerIdsMap = QMap<QString, DeviceProperties>{};
         foreachDevices(TEXT("USB"),
-            [&containerId, &devInfo] (DeviceProperties deviceProperties) -> void {
-                if (deviceProperties.containerId == containerId
-                        && !deviceProperties.instanceId.contains("MI_")) {
-                    devInfo.devId = extractDevPidVidInfo(deviceProperties.instanceId);
-                    devInfo.usbPortPath = extractUsbPortPath(deviceProperties.locationPath);
+            [&usbDevicesByContainerIdsMap] (DeviceProperties deviceProperties) -> void {
+                if (!deviceProperties.instanceId.contains("MI_")) {
+                    usbDevicesByContainerIdsMap.insert(deviceProperties.containerId,
+                                                       std::move(deviceProperties));
```

Comment:
This `std::move` is useless since there's no `QMap::insert` overload which accepts `rvalue` ref:
![image](https://user-images.githubusercontent.com/44399142/64062804-44e7a000-cbf4-11e9-88b1-5dae138d912c.png)

Reply:
Ok, thanks